### ISSUE TITLE
Add removeAttestationSigner to ReleaseGold contract

### ISSUE THREAD

**Issue by pahor167** [celo-org/celo-monorepo#10148]
TLDR:

- There is no wrapper function for removeAttestationSigner() in [ReleaseGold.sol](https://github.com/celo-org/celo-monorepo/blob/master/packages/protocol/contracts/governance/ReleaseGold.sol) from [Accounts.sol](https://github.com/celo-org/celo-monorepo/blob/29fe6f7b6e01f683c54d7f99eabfaca02c09fe40/packages/protocol/contracts/common/Accounts.sol#L732)
- ASv1 operators that use ReleaseGold (i.e. former cLabs employees) can authorise but not (!) deauthorise their attestation signers
- We’re sunsetting ASv1, so ReleaseGold validators will want to call removeAttestationSigner() to complete the sunsetting.

https://gist.github.com/0xarthurxyz/cad6535d5ba69f749804b89e832c8034

**Comment by pahor167** [comment]
Eng team decided against patching this small issue because:

- no actual behaviour change (after Attestations.sol will be made read-only)
- best practice: “minimise changes to smart contracts”
